megamek/src/megamek/common/units/BipedMek.java

```diff
@@ -191,8 +191,11 @@ public int getWalkMP(MPCalculationSetting mpCalculationSetting) {
                     minReduction = (int) Math.ceil(mp / 2.0);
                     maxReduction = (int) Math.ceil(minReduction / 2.0);
 
+                    // playtest 2 update
+                    /*
                     if (minReduction < 2) {minReduction = 2;}
-                    if (maxReduction < 2) {maxReduction = 2;}
+                    if (maxReduction < 2) {maxReduction = 2;}              
+                     */
```

Comment:
Please just remove the code rather than commenting it out.  If we need to get it back, we have the git commit history.